pages/about.css

```diff
@@ -0,0 +1,70 @@
+
+.SCA-about {
+
+}
+
+.titleschool {
+  color: grey;
+  text-decoration: bold;
+}
+.titleapp {
+  color: grey;
+  font-size: 28px;
+}
+
+  a:active, a:visited, a:link {
```

Comment:
We should have it in index.css as discussed.
They are used for each a in each page.

Reply:
ok